osgi-dependencies/proxy/src/main/java/com/cognifide/aet/proxy/RestProxyServer.java

```diff
@@ -95,18 +94,10 @@ public void setCaptureContent(boolean captureContent) {
   }
 
   @Override
-  public void addHeader(String name, String value) {
+  public void addHeader(String name, String value, Boolean override) {
     CloseableHttpClient httpClient = HttpClients.createSystem();
     try {
-      URIBuilder uriBuilder = new URIBuilder().setScheme(HTTP).setHost(server.getAPIHost())
-          .setPort(server.getAPIPort());
-      // Request BMP to add header
-      HttpPost request = new HttpPost(uriBuilder.setPath(
```

Comment:
According to https://github.com/lightbody/browsermob-proxy#rest-api docs:
`POST    /proxy/[port]/headers`
> Set and override HTTP Request headers

Do those changes mean, that this is not true? There is even an example of setting the `User-Agent` header.

Reply:
Yes, this doesn't work anymore. You can't use this to change some headers like User-agent. Using this you can only add new header or override header that you add before.